### DISCUSSION TITLE
I want documentation (please).

### DISCUSSION THREAD

**Discussion by IHateGameDev** [SpaRcle-Studio/SREngine#260] [Q&A] [answered]
Hello world! Finally, I waited for the release of this engine on Linux X11. But I faced one rather significant problem for me, namely, I don't understand what to do at all.
![idk](https://github.com/SpaRcle-Studio/SREngine/assets/172628112/3a289727-608f-4f56-b5b2-616cf87aff01)
(I didn't find the documentation, but maybe I just didn't look for it well. :( ) P
lease give a link or write something like "We didn't do the documentation because we're working on engine systems." or something like that.

**Comment by innerviewer** [comment, marked as answer]
Hello. Thanks for the feedback! We are glad that someone is interested in our engine. 

There is no documentation available at the moment. In fact, we are currently working on the engine's systems and UI. 
After we get rid of the ImGUI and fully move to our framework, I'll start working on the documentation. 

Anyway, could you please give some feedback on the stability of the engine on your platform? 
The main developer uses Windows and I'm the only one adding Linux compatibility. 
However, since I'm using i3wm, I don't do much manual window resizing. I'd also like to know if there are any problems with that.

**Reply by IHateGameDev** [reply]
### Hello again!
Of course, it's a pity that there is no documentation yet, but I can wait.

### Stable
I can't say for sure how stable the engine is, because I didn't spend five hours on it, but I can highlight the moments I encountered.

   **Moments:**

1. - At the first to third entry into the engine, it immediately flew out of it (in the logs it wrote something about the Vulkan error, but I did not save the exact information).
2. - The screenshot shown in my question had the text. When I tried to move it in the scene, the Transform changed, but its visual position relative to the camera did not change in any way.
3. - When I opened the Assets window and tried to attach it to any part of the working area, I did not succeed, if you did not implement it, then think about implementing this.

### END
This is all I experienced... I will work with the engine again and write (probably in a separate discussion) what difficulties I will face.
**P.s.** I am very pleased that you (created) quickly answered my question. **_I also wanted to ask if you support discussions in Russian._**

**Reply by innerviewer** [reply]
> At the first to third entry into the engine, it immediately flew out of it (in the logs it wrote something about the Vulkan error, but I did not save the exact information).
> The screenshot shown in my question had the text. When I tried to move it in the scene, the Transform changed, but its visual position relative to the camera did not change in any way.

Thanks again for your feedback. 
This is very unusual for us, so I'll think about some sort of bug tracking for our team. 
We'll try to fix it and keep you updated. 

> When I opened the Assets window and tried to attach it to any part of the working area, I did not succeed, if you did not implement it, then think about implementing this.

At the moment this logic works fine on our systems. If there are problems with docking, I'm 90% sure it's ImGUI itself. 
Anyway, I'll try to recreate this on my machine. 

> I also wanted to ask if you support discussions in Russian.

We try to keep our GitHub in English only, and such discussions might help someone else as well. 
But if you want to discuss something in Russian, feel free to join our Discord server (there's a badge in the readme).
